### PR TITLE
[WEB] DATA FOR RESEARCHERS

### LINKED ISSUE #294: Data for Researchers Implementation
### Purpose

We want to have a page which allows users to download our collected data from voyages. 

### Description

1. Create a Drop down component using Accordion: https://mui.com/material-ui/react-accordion/. This is where the users will download the data. 
2. Create a container for Polaris (We were thinking of having individual containers for each article). Refer to UBCSailbot/website#61 on how to create a container. 
     - Pass in three props for the `image`, `header`, and `desc`.
    
image

### Resources

- Example Template: https://github.com/webexpe13/blog-template-using-nextjs-typescript-tailwindcss/tree/main

### PR DESCRIPTION
**Pull request by JordanChen123**
### Description

- Resolves #294 

### Verification

- [ ] 

### Resources

-

### CODE REVIEW

File: src/website/views/components/Header/header.module.css
Comment (jahn18) [inline src/website/views/components/Header/header.module.css:1]: I noticed the text "UBC Sailbot" on the header got centered: 

image

Could we place the text on the left side as we had previously? The position of the info button is perfectly fine.

**Review by jahn18 — CHANGES_REQUESTED** [review]
After addressing the comment, please also to make sure to run `npm format` and `npm lint`!

File: src/website/views/components/Header/header.module.css
Comment (JordanChen123) [reply src/website/views/components/Header/header.module.css:1]: Done !

**Review by jahn18 — APPROVED** [review]
Thanks Jordan! 😄 Looks good

**Pull request merged**